nodebuilder/daser/config.go

```diff
@@ -0,0 +1,76 @@
+package daser
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	ErrNoSamplingRange = errors.New("nodebuilder/daser: misconfiguration error, sampling range cannot be 0")
+	ErrNoConcLimit     = errors.New("nodebuikder/daser: misconfiguration error, concurrency limit cannot be 0")
+	ErrNoGenesisHeight = errors.New("nodebuilder/daser: misconfiguration error, genesis height cannot be 0")
+)
```

Comment:
This error does not have to be exported and extracted in var, or do you see them being somehow? Because I don't